notebooks/image_models/solutions/create_tfrecords_at_scale.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %%writefile create_tfrecords.py
</pre><p>+1 I'd break this code in chunk to be run it in cells (rather than as a script) prefixed by explanations in markdown.</p><p>As it is, it seems too bulky for a first dive into dataflow.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/224/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>